### PR TITLE
CDAP-19637: display rejected requests on hdf side

### PR DESCRIPTION
**Pull request by mrahanjam**
# Display rejected requests on hdf side

## Description
This PR changes `Pending Requests` on hdf side into `Tethering Requests` and displays `rejected` requests as well so that the admin can delete them.

## PR Type
- [ ] Bug Fix
- [X] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19637](https://cdap.atlassian.net/browse/CDAP-19637)

## Test Plan
N/A

## Screenshots

Screen Shot 2022-09-29 at 10 24 58 AM

### CODE REVIEW

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx [hunk 1]
```diff
@@ -124,6 +134,7 @@ const TetheringTable = ({
           key={i}
         >
           <GridCell>{isFirst ? icon : ''}</GridCell>
+          <GridCell>{isFirst ? capitalizeFirstLetter(tetheringStatus) : ''}</GridCell>
```
Comment (njbriggs) [inline app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx]: Does `tetheringStatus` come for the API? If it needs to be formatted for display, would we be better off I18ning the display strings?

**Comment by njbriggs** [comment]
One question, everything else LGTM.

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx (hunk 1 above)
Comment (mrahanjam) [reply app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx]: Yes it comes from the API. Let me look into that.

**Comment by mrahanjam** [comment]
> One question, everything else LGTM.

Addressed

**Review by sumengwang — APPROVED** [review]

**Review by seanfeld — APPROVED** [review]

**Review by de-lan — APPROVED** [review]
lgtm

**Review by njbriggs — APPROVED** [review]

**Pull request merged**